frontend/dockerfile/docs/reference.md

```diff
@@ -1316,79 +1286,87 @@ doesn't support authentication.
 - If `<dest>` doesn't exist, it's created, along with all missing directories
   in its path.
 
-### Verifying a remote file checksum `ADD --checksum=<checksum> <http src> <dest>`
```

Comment:
Note to reviewer: I renamed this section (and the "Add a Git repository" one) to `ADD --flag` to keep consistency with the others